### PR TITLE
CategoryControllerTests

### PR DESCRIPTION
**Pull request by Grzes88**

### CODE REVIEW

File: src/MyShop.Infrastructure/DAL/MSqlOptions.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿namespace MyShop.Infrastructure.DAL;
 
-internal sealed class MSqlOptions
+public sealed class MSqlOptions
```
Comment (szymio11) [inline src/MyShop.Infrastructure/DAL/MSqlOptions.cs]: zostaw internal i daj visibility

Comment (szymio11) [reply src/MyShop.Infrastructure/DAL/MSqlOptions.cs]: <InternalsVisibleTo Include="MyShop.Tests.Integration\MyShop.Tests.Integration.csproj" />

Comment (szymio11) [reply src/MyShop.Infrastructure/DAL/MSqlOptions.cs]: `<InternalsVisibleTo Include="MyShop.Tests.Integration\MyShop.Tests.Integration.csproj" />`

Comment (szymio11) [reply src/MyShop.Infrastructure/DAL/MSqlOptions.cs]: taki coś powinno stykać

File: tests/MyShop.Tests.Integration/Controllers/CategoryControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    private readonly TestDatabase _testDatabase;
+
+    public CategoryControllerTests()
+        => _testDatabase = new TestDatabase();
+
+    [Fact]
+    public async Task Post_Category_Should_Return_NoContent_204_Status_Code()
+    {
+        //Act
+        var command = new CreateCategory("Buty");
```
Comment (szymio11) [inline tests/MyShop.Tests.Integration/Controllers/CategoryControllerTests.cs:22]: to jest assert

File: src/MyShop.Infrastructure/DAL/MSqlOptions.cs (hunk 1 above)
Comment (Grzes88) [reply src/MyShop.Infrastructure/DAL/MSqlOptions.cs]: dodalem te <InternalsVisibleTo Include="MyShop.Tests.Integration\MyShop.Tests.Integration.csproj" /> referencje  do Myshop.infrastructure  i nie ma dostepu do klasy msqloptions.

File: src/MyShop.Infrastructure/DAL/MSqlOptions.cs [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ﻿namespace MyShop.Infrastructure.DAL;
 
+
```
Comment (szymio11) [inline src/MyShop.Infrastructure/DAL/MSqlOptions.cs]: po co to?

Comment (Grzes88) [reply src/MyShop.Infrastructure/DAL/MSqlOptions.cs]: nie wiem :D

**Review by szymio11 — APPROVED** [review]

**Pull request merged**